storage/fs.go

```diff
@@ -161,16 +156,11 @@ func (f *Filesystem) readDir(ctx context.Context, src *url.URL, ch chan *Object)
 	}
 }
 
-func (f *Filesystem) walkDir(ctx context.Context, src *url.URL, isRecursive bool) <-chan *Object {
+func (f *Filesystem) walkDir(ctx context.Context, src *url.URL) <-chan *Object {
 	ch := make(chan *Object)
 	go func() {
 		defer close(ch)
 
-		if !isRecursive {
-			f.readDir(ctx, src, ch)
```

Comment:
Remove fs.readDir() method. It is not used anymore.

Reply:
done.